### PR TITLE
Update vendor dependencies in go.mod

### PR DESCRIPTION
**Pull request by shajmakh**
bump pao version:
-Add latency test cases
-Add RT kernel check before checking rcu existence

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Docker Image CI
```
Comment (Tal-or) [inline .github/workflows/main.yml:1]: Why is this file needed?

Comment (shajmakh) [reply .github/workflows/main.yml:1]: Not sure why , it was created upon executing make deps-update. I'll try to redo the process .

Comment (shajmakh) [reply .github/workflows/main.yml:1]: After doing the process again, I see that the main.yaml gets created & added & committed automatically upon "make deps-update". Should I try & revert this file commit? @MarSik  please advise.

Comment (shajmakh) [reply .github/workflows/main.yml:1]: ci tests passed without the main.yaml, will close this one since https://github.com/openshift-kni/cnf-features-deploy/pull/680 already passed.

**Comment by shajmakh** [comment]
refer to this instead: https://github.com/openshift-kni/cnf-features-deploy/pull/680
